src/js/disability-benefits/components/ClaimDetailLayout.jsx

```diff
@@ -15,13 +15,14 @@ export default class ClaimDetailLayout extends React.Component {
 
     let content;
     if (!loading) {
+      const claimsLink = `your-claims${claim.attributes.open ? '' : '/closed'}`;
```

Comment:
The following scenario was causing unexpected behavior
1. User navigates to page 2 of closed claims
2. User clicks on a claim
3. User clicks link to 'Your Claims'
4. User is on page 2 of open claims

I think preserving the pagination on back navigation is important UX, which requires maintaining the pagination state between mounts of YourClaimsPage, so this change mimics going back from a claim